ironmon_tracker/Program.lua

```diff
@@ -684,3 +684,14 @@ function Program.getBagStatusItems()
 
 	return statusItems
 end
+
+function Program.playerHasMove(moveName)
```

Comment:
Can this be placed in Utils.lua? I feel like it is better scoped as a utility function rather than a main program function.